test/specs/equipment.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from "@wdio/globals";
+import MainPage from "../pageobjects/main.page.ts";
+import UnitPage from "../pageobjects/unit.page.ts";
+import ProductsPage from "../pageobjects/products.page.ts";
+import * as testData from "../data/testdata.json";
+
+describe("Rentzila", () => {
+  it('C213- Checking ""Спецтехніка"" section on the main page', async () => {
```

Comment:
@Anastas11aZakharova This test case failed for me.

Reply:
This is expected because of the data issue in the dev environment. Multiple categories do not have actual units assigned to them, hence the check fails, please see screenshot for an example of such case.

Could add a check that if no items are found, skip the check on the unit page. But then it would "hide" the actual data issue and will not align to the test case.
![Screenshot 2024-10-05 151610](https://github.com/user-attachments/assets/4c348f3c-8338-40cb-a201-2dd8a8807f23)